### DISCUSSION TITLE
Not seeing my program in the simulator

### DISCUSSION THREAD

**Discussion by Cruisechef** [zepp-health/zeppos-forum#387] [User Feedback]
Try as I may,  I cannot get the program to appear in the simulator as shown in the tutorials.   What step am I missing?

I am using:
zeus init
zeus create
Zeus dev

Then manually opening the simulator.  and nothing.. Zip

![Screenshot 2024-11-03 3 58 21 PM](https://github.com/user-attachments/assets/3d6728ff-4bac-4fba-9e18-96454ce0b2be)

**Comment by silver-zepp** [comment]
You won't see anything there until you run the Emulator and zeus syncs the project. Start the simulator from a terminal `> path/to/simulator.exe` (win example) and observe the logs. It's most likely some permission or a missing library on your system. 

You can try closing the sim, removing the `C:\Users\{username}\.zepp` folder, and then trying again.

**Comment by Cruisechef** [comment]
so open the simulator and then run the **zeus dev** command?   In that order?

then Just wait?

**Reply by Cruisechef** [reply]
when you say emulator I assume you mean the simulator?

**Comment by MyoungXUE** [comment]
First, you'll need to execute the `zeus create` command to initiate the creation of a Zepp OS program.

After that, you can navigate to the project directory and execute the `zeus dev` command to preview the program within the Emulator.

Please adhere strictly to the quick start guide provided in the documentation.

https://docs.zepp.com/docs/guides/quick-start/create/

**Comment by Cruisechef** [comment]
I have done that. Dozens of times.  Still not working
